npp/FHIR.Npp.Validator.pas

```diff
@@ -730,8 +730,9 @@ procedure TFHIRPluginValidatorContextR4.checkClient;
       FServer.Free;
     FServer := Factory.makeClient(self.link, FUrl, fctWinInet, ffJson, 5000) as FHIR.R4.Client.TFhirClient4;
     FCapabilityStatement := FServer.conformance(true);
-    if FCapabilityStatement.fhirVersion <> FHIR_VERSIONS[factory.version] then
-      raise EFHIRException.create('Terminology Server / Plug-in Version mismatch ('+FCapabilityStatement.fhirVersion+' / '+CODES_TFHIRVersion[factory.version]+')');
+    if CODES_TFhirFHIRVersionEnum[FCapabilityStatement.fhirVersion] <> FHIR_VERSIONS[factory.version] then
```

Comment:
Please check this part as I'm not sure about it. The original version did not compile (incompatible types).